app/components/widgets/forms/radio-button.js

```diff
@@ -1,31 +1,35 @@
+import { tracked } from '@glimmer/tracking';
+import classic from 'ember-classic-decorator';
+import { attributeBindings, tagName } from '@ember-decorators/component';
+import { observes } from '@ember-decorators/object';
+import { computed } from '@ember/object';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['computed' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)